docs/src/man/comparisons.md

```diff
@@ -204,7 +204,7 @@ df <- tibble(grp = rep(1:2, 3), x = 6:1, y = 4:9,
 | Rename columns           | `rename(df, x_new = x)`        | `rename(df, :x => :x_new)`             |
 | Pick columns             | `select(df, x, y)`             | `select(df, :x, :y)`                   |
 | Pick & transform columns | `transmute(df, mean(x), y)`    | `select(df, :x => mean, :y)`           |
-| Pick rows                | `filter(df, x >= 1)`           | `filter(:x => >=(1), df)`              |
+| Pick rows                | `filter(df, x >= 1)`           | `subset(df, :x => ByRow(>=(1)))`       |
```

Comment:
Maybe we should also show `x -> x >= 1`? It may be simpler to grasp for many users. Or does that take too much space?

Reply:
changed. I also added a description why `>=(1)` is preferred from the perspective of compilation cost.